src/library/scala/util/Either.scala

```diff
@@ -628,7 +625,6 @@ object Either {
      *  Right(12).left.filter(_ > 10) // None
      *  }}}
      */
-    @deprecated("Use `filterToOption`, which more accurately reflects the return type", "2.13.0")
```

Comment:
That one should still stay deprecated, no?

Reply:
Right. Fixed.